### ISSUE TITLE
Methods provided by Any may shadow methods provided by a Perl 5 class

### ISSUE THREAD

**Issue by hoelzro** [niner/Inline-Perl5#32]
For example:

```perl
package RobTest;

use strict;
use warnings;

sub new {
    my ( $class ) = @_;

    return bless {}, $class;
}

sub print {
    print "I'm a RobTest\n";
}

sub print_remix {
    print "I'm a RobTest (remix!)\n";
}

1;
```

```perl6
use v6;

use Inline::Perl5;
use RobTest:from<Perl5>;

my $rt = RobTest.new;
$rt.print;
$rt.print_remix;
```

`$rt.print` invokes `Any.print`, but `$rt.print_remix` dispatches correctly.

**Comment by vendethiel** [comment]
I think that's already been an issue in the past also.

**Comment by niner** [comment]
Fixed in commit 3acaa28bc7979b68eda6348995a80e793c1c739b

The issue was not really about Any's methods but about Mu's. So we now explicitly wrap the ones we should be able to get away with.
